src/config.ts

```diff
@@ -40,7 +40,7 @@ const configSchema = Type.Object(
      */
     nullableName: Type.String({ default: "__nullable__" }),
     /**
-     * Wethter to allow recursion in the generated schemes (enabling reduces code size)
+     * Wether to allow recursion in the generated schemes (enabling reduces code size)
```

Comment:
Nice catch, thanks!

Reply:
Still spelt it incorrectly, oops... Should be "whether"